app/services/request_update_service.rb

```diff
@@ -150,7 +150,6 @@ def notify_request
 
   # complete the external approval process if configured
   def finish_request(decision)
-    return if request_completed?(decision)
```

Comment:
Is there no spec covering the removal of this line or is it being somehow tested within the `jbpm_process_service_spec` that you changed?

Reply:
I added a spec in request_update_service_spec.rb